setup.cfg

```diff
@@ -1,3 +1,74 @@
+[metadata]
+name = borgbackup
+author = The Borg Collective (see AUTHORS file)
+description = Deduplicated, encrypted, authenticated and compressed backups
+url = https://github.com/borgbase/vorta
+keywords =
+  backup
+  borgbackup
+# List of classifiers: https://pypi.org/pypi?%3Aaction=list_classifiers
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Console
+    Intended Audience :: System Administrators
+    License :: OSI Approved :: BSD License
+    Operating System :: POSIX :: BSD :: FreeBSD
+    Operating System :: POSIX :: BSD :: OpenBSD
+    Operating System :: POSIX :: BSD :: NetBSD
+    Operating System :: MacOS :: MacOS X
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Topic :: Security :: Cryptography
+    Topic :: System :: Archiving :: Backup
+platforms = Linux, MacOS X, FreeBSD, OpenBSD, NetBSD
+long_description = file: README.rst
+license = BSD
+license_file = LICENSE
+project_urls =
+    Bug Tracker = https://github.com/borgbackup/borg/issues
+    Documentation = https://borgbackup.readthedocs.io
+    Source Code = https://github.com/borgbackup/borg
+
+[options]
+packages = find:
+package_dir =
+    =src
+# See also the MANIFEST.in file.
+# We want to install all the files in the package directories...
+include_package_data = true
```

Comment:
this used to be right above exclude_package_data, so comments could be understood together.